src/labeled_signal.jl

```diff
@@ -158,8 +165,18 @@ data represented by the _signal_.
 If any label span is not entirely contained within the corresponding signal
 span, this will throw an ArgumentError.
 """
-function label_signals(signals, annotations; groups=:recording, kwargs...)
-    labels_table = labels_to_samples_table(annotations; groups, kwargs...)
+function label_signals(signals, annotations;
+                        groups=:recording,
+                        labels_column,
+                        epoch,
+                        encoding,
+                        roundto=nothing)
+    labels_table = labels_to_samples_table(annotations;
+                                            groups,
+                                            labels_column,
+                                            epoch,
+                                            encoding,
+                                            roundto)
```

Comment:
The [YAS style guide](https://github.com/jrevels/YASGuide#linealignmentspacing-guidelines) is not very clear on what the expectations here are... it only says

>If an argument list extends to multiple lines, align new lines to the parenthesis/bracket/etc. that started the argument list.

it doesn't say if args should exhaust the line limit before continuing on the next line or if it's preferable to line break at each arg....I opted for copying what I saw elsewhere in the package

Reply:
yeah, it's not the proscriptive